improver/calibration/rainforest_calibration.py

```diff
@@ -870,8 +867,6 @@ def _calculate_error_probabilities(
                 If an unsupported model object is passed. Expects lightgbm Booster, or
                 treelite_runtime Predictor (if treelite dependency is available).
         """
-        from treelite_runtime import DMatrix
```

Comment:
I think the method `_calculate_error_probabilities` is now exactly the same as the one for the super class, and can be deleted.

Reply:
I've done this in 2dff508.